chord_metadata_service/phenopackets/serializers.py

```diff
@@ -60,7 +63,7 @@ class Meta:
 
 class PhenotypicFeatureSerializer(GenericSerializer):
     always_include = (
-        "negated",
+        "excluded",
```

Comment:
Why is `excluded` always included? I realize this is a just a rewrite of the old code, it's not clear to me if this is still needed or not.

Reply:
Yeah this is just a name change but I found it strange too.

From the [specs](https://phenopacket-schema.readthedocs.io/en/2.0.0/phenotype.html#excluded):
> This element is a flag to indicate whether the phenotype was observed or not. The default is ‘false’, in other words the phenotype was observed. Therefore it is only required in cases to indicate that the phenotype was looked for, but found to be absent.

Reply:
This seems like a design decision that was made a long time ago, but the reasoning for it isn't clear to me, it seems like odd treatment for an optional field. This value appears in one part of the front end but as far as I know none of our data has ever used it (ie, it's always `false`). @davidlougheed?